src/providers/trusted_service/context/error.rs

```diff
@@ -0,0 +1,203 @@
+// Copyright 2020 Contributors to the Parsec project.
+// SPDX-License-Identifier: Apache-2.0
+use log::error;
+use psa_crypto::types::status::{Error as PsaError, Status as PsaStatus};
+use std::error::Error as ErrorTrait;
+use std::fmt;
+
+/// Wrapper over types of error that the TS Context might return
+#[derive(Debug, PartialEq, Copy, Clone)]
+pub enum Error {
+    /// errors coming from the Crypto TS service, associated with performed operations.
+    PsaCryptoError(PsaError),
```

Comment:
I would say the `Error` suffix is not needed for the variants 😋

Reply:
Fixed locally - will upstream in a bit